From 1c64fb9b63eca3534a7dcfac398554d974b9cb67 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:12:11 -0400
Subject: [ethernet] i40evf: use correct format for printing MAC addresses

Message-id: <1409663591-32531-89-git-send-email-sassmann@redhat.com>
Patchwork-id: 90565
O-Subject: [RHEL7.1 PATCH 088/148] i40evf: use correct format for printing MAC addresses
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

The correct format is %pM, not %pMAC.

Change-ID: Idb335723a966fe56db3a72b9c07c08ca66f9db3c
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Tested-by: Sibai Li <sibai.li@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit b34f90e7040d8c20ceb51b26359ea6014a707921)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 5fd60dd..590a9a4 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -2058,7 +2058,7 @@ static void i40evf_init_task(struct work_struct *work)
  netdev->hw_features &= ~NETIF_F_RXCSUM;
 
  if (!is_valid_ether_addr(adapter->hw.mac.addr)) {
-  dev_info(&pdev->dev, "Invalid MAC address %pMAC, using random\n",
+  dev_info(&pdev->dev, "Invalid MAC address %pM, using random\n",
     adapter->hw.mac.addr);
   random_ether_addr(adapter->hw.mac.addr);
  }
@@ -2116,7 +2116,7 @@ static void i40evf_init_task(struct work_struct *work)
 
  netif_tx_stop_all_queues(netdev);
 
- dev_info(&pdev->dev, "MAC address: %pMAC\n", adapter->hw.mac.addr);
+ dev_info(&pdev->dev, "MAC address: %pM\n", adapter->hw.mac.addr);
  if (netdev->features & NETIF_F_GRO)
   dev_info(&pdev->dev, "GRO is enabled\n");
 
-- 
1.7.1